matrix/colors.py

```diff
@@ -88,6 +88,11 @@ def from_input_line(cls, line):
         substrings = []  # type: List[FormattedString]
         attributes = DEFAULT_ATTRIBUTES.copy()
 
+        # If this is false, only IRC formatting characters will be parsed.
+        # (If the option is unset, assume True.)
+        do_markdown = G.CONFIG.look.markdown_input \
```

Comment:
I don't think that the option can be unset considering that it's a boolean option with a defined default value, did you observe that it is unset somewhere?

Reply:
Yes, when running tests. :)

Reply:
Ah ok, I think we solve this by creating a mock config over [here](https://github.com/poljar/weechat-matrix/blob/master/matrix/_weechat.py#L29) instead of adding checks to the code.

Reply:
Yes I already added [an entry](https://github.com/poljar/weechat-matrix/pull/229/files#diff-1f0f93ab3ac3ffabb8f79ac4c18f866aR65) to make the tests even run, but I set it to None because all other mock values did. Setting it to True there would allow removing this check from the main code.

Reply:
Indeed, please don't ask me why all of them are `None` :sweat_smile:

Reply:
Applied as promised :)